master/docs/tutorial/tour.rst

```diff
@@ -259,7 +259,7 @@ To use this you will need to install an additional package or two to your virtua
   cd
   cd tmp/buildbot
   source sandbox/bin/activate
-  easy_install pycrypto
+  easy_install cryptography
```

Comment:
could you replace easy_install with pip ?
there is a need for a pip install -U pip in the first place